src/sections/Meshmap/meshmap.style.js

```diff
@@ -2,15 +2,21 @@ import styled from "styled-components";
 
 const MeshmapWrapper = styled.div`
   
-
+  .supscript{
+    /* background-color: red; */
```

Comment:
Remove this, don't comment out when not required directly remove it.

Reply:
ok, I will do that.